src/importer/tflite/opcode.def

```diff
@@ -64,3 +64,4 @@ DEFINE_OPCODE(GATHER_ND)
 DEFINE_OPCODE(ONE_HOT)
 DEFINE_OPCODE(SQUARED_DIFFERENCE)
 DEFINE_OPCODE(LOG_SOFTMAX)
+DEFINE_OPCODE(SPLIT)
```

Comment:
SPLIT op should be added into docs/tflite_ops.md too.

Reply:
> SPLIT op should be added into docs/tflite_ops.md too.

Done!